lib/plugins/aws/package/compile/events/alb/lib/validate.js

```diff
@@ -44,12 +54,38 @@ module.exports = {
         }
       });
     });
+    this.validatePriorities(events);
 
     return {
       events,
     };
   },
 
+  validateListenerArnAndExtractAlbId(event, functionName) {
```

Comment:
Is a nit, but the name of the function is a little bit misleading since it returns / extracts 2 values.

Furthermore we can pass the `listenerArn` rather than the `event` since the only processing we do with the `event` is to extract the `listenerArn`.